tests/e2e/testcases/blockchain/configure/suite.go

```diff
@@ -0,0 +1,470 @@
+// Copyright (C) 2025, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package configure
+
+import (
+	"fmt"
+	"os"
+	"path"
+
+	"github.com/ava-labs/avalanche-cli/pkg/constants"
+	"github.com/ava-labs/avalanche-cli/tests/e2e/commands"
+	"github.com/ava-labs/avalanche-cli/tests/e2e/utils"
+
+	ginkgo "github.com/onsi/ginkgo/v2"
+	"github.com/onsi/gomega"
+)
+
+const (
+	defaultRPCTxFeeCap = 100
+	newRPCTxFeeCap1    = 101
+	newRPCTxFeeCap2    = 102
+	acpSupport1        = 111
+	acpSupport2        = 112
+	node1CertPath      = "tests/e2e/assets/node1/staker.crt"
+	node2CertPath      = "tests/e2e/assets/node2/staker.crt"
+	node1TLSPath       = "tests/e2e/assets/node1/staker.key"
+	node2TLSPath       = "tests/e2e/assets/node2/staker.key"
+	node1BLSPath       = "tests/e2e/assets/node1/signer.key"
+	node2BLSPath       = "tests/e2e/assets/node2/signer.key"
+	node1ID            = "NodeID-GWPcbFJZFfZreETSoWjPimr846mXEKCtu"
+	node2ID            = "NodeID-P7oB2McjBGgW2NXXWVYjV8JEDFoW9xDE5"
+)
+
+func checkBlockchainConfig(
```

Comment:
can we add comments on what this function is checking for?

Reply:
seems that we only check RPCTxFeeCap, is there an exhaustive list of the properties we can modify with blockchainConfig? 

It will help users to know the full list of items they can modify with blockchain configure command

Reply:
I would rename it something like `AssertBlockchainConfigContainsSomething` or something similar, because this is not just util function, but an actual assertion.

Reply:
is general terms, config is defined by the vm, it can be anything so for custom vms we can't check too much or provide too much info. this e2e is based on subnet evm, for this case, the config options are on:

https://github.com/ava-labs/subnet-evm/blob/master/plugin/evm/config/config.go

Reply:
we are expecting users to know which are the config options, but not close to add some doc or ux on this for subnet-evm